src/components/cc-invoice-list/cc-invoice-list.js

```diff
@@ -97,25 +110,28 @@ export class CcInvoiceList extends LitElement {
       ? maxFromStrings(processedInvoicesYears)
       : this._yearFilter;
 
-    const filteredProcessedInvoices = processedInvoices.filter((i) => getYearAsString(i.emissionDate) === yearFilter);
+    const filteredProcessedInvoiceTableState = {
+      type: 'loaded',
+      invoices: processedInvoicesTableState.invoices.filter((i) => getYearAsString(i.emissionDate) === yearFilter),
+    };
 
-    const hasYearSelector = filteredProcessedInvoices.length > 0 && yearChoices.length > 1;
+    const hasYearSelector = filteredProcessedInvoiceTableState.invoices.length > 0 && yearChoices.length > 1;
 
     return this._renderView(html`
```

Comment:
note: not a great fan of the "reversed renderView" but it was already there so OK.